docs/source/api/passes.rst

```diff
@@ -42,6 +42,12 @@ OnnxFloatToFloat16
 --------------------
 .. autoconfigclass:: olive.passes.OnnxFloatToFloat16
 
+.. _onnx_io_float16_to_float32:
+
+OnnxIOFloat16ToFloat32
+--------------------
```

Comment:
Doc build test is complaining about this line being shorter than the title.

Reply:
Fixed